packages/EasyCore/src/Bridge/Symfony/Validator/Constraints/Decimal.php

```diff
@@ -34,13 +35,16 @@ final class Decimal extends Constraint
      */
     public $minPrecision;
 
-    public function __construct($options = null)
+    public function __construct($options = null, int $minPrecision = null, int $maxPrecision = null)
```

Comment:
Pls add parameters as discussed in Slack.

Reply:
done